std/stdio.d

```diff
@@ -3427,6 +3425,24 @@ void main()
     assert(e && e.msg == "Attempting to write to closed File");
 }
 
+version(StdStressTest)
+{
+    // issue 15768
+    @system unittest
+    {
+        import std.parallelism : parallel;
+        import std.range : iota;
+
+        auto deleteme = testFilename();
+        stderr = File(deleteme, "w");
+
+        foreach(t; 1_000_000.iota.parallel)
```

Comment:
std/stdio.d:3439:        foreach(t; 1_000_000.iota.parallel)
posix.mak:544: recipe for target 'style_lint' failed